image/rhel/fetch-stackrox-data.sh

```diff
@@ -15,7 +15,7 @@ fetch_stackrox_data() {
     local download_dir
     download_dir="$(mktemp -d --tmpdir external-networks.XXXXXXXXXX)"
     local latest_prefix
-    latest_prefix="$(curl --fail https://definitions.stackrox.io/external-networks/latest_prefix)"
+    latest_prefix="$(curl --fail https://definitions.stackrox.io/external-networks/latest_prefix | sed 's/ /%20/g')"
```

Comment:
I've seen some folks around here prefer bash:
```suggestion
    latest_prefix="$(curl --fail https://definitions.stackrox.io/external-networks/latest_prefix)"
    latest_prefix="${latest_prefix// /%20}"
```